test/Datadog.Trace.Tests/DogStatsDTests.cs

```diff
@@ -128,7 +123,7 @@ private static IImmutableList<MockTracerAgent.Span> SendSpan(bool tracerMetricsE
                     Thread.Sleep(5);
                 }
 
-                spans = agent.WaitForSpans(1);
+                spans = agent.WaitForSpans(1, timeoutInMilliseconds: 1000000);
```

Comment:
One thousand second timeout? That seems a long time to wait.
What was the thought here?

Reply:
I guess I tested something then forgot to revert it